From e77ff6c3ba76192787122f2b768abd2a81e8f443 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Thu, 14 Nov 2019 10:38:52 -0800
Subject: [PATCH] Add IDL compiler dependencies

---
 debian/control.em | 2 +-
 package.xml       | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 6d7b65c..a1d3b9d 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -2,7 +2,7 @@ Source: @(Package)
 Section: misc
 Priority: optional
 Maintainer: @(Maintainer)
-Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(BuildDepends))
+Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(['default-jdk', 'maven'] + BuildDepends))
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
diff --git a/package.xml b/package.xml
index 9d0980d..5b4b661 100644
--- a/package.xml
+++ b/package.xml
@@ -12,6 +12,8 @@
     <url type="repository">https://github.com/eclipse-cyclonedds/cyclonedds</url>
 
     <buildtool_depend>cmake</buildtool_depend>
+    <buildtool_depend>java</buildtool_depend>
+    <buildtool_depend>maven</buildtool_depend>
     <depend>openssl</depend>
     <test_depend>libcunit-dev</test_depend>
     <doc_depend>python3-sphinx</doc_depend>
-- 
2.23.0

