From 3e53a1c5a2b88f83eb3697a3414e33d02473d261 Mon Sep 17 00:00:00 2001
From: Michael Carroll <michael@openrobotics.org>
Date: Wed, 23 Oct 2019 22:47:46 -0500
Subject: [PATCH] Don't build IDLC

---
 debian/rules.em | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index aa010fa..d8f2309 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -26,7 +26,9 @@ override_dh_auto_configure:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_configure -- -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)"
+	dh_auto_configure -- \
+					-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
+					-DBUILD_IDLC=OFF
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.17.1

